doc/sphinx/reactors.rst

```diff
@@ -195,3 +181,230 @@ temperature:
+
+3. *Optional:* Set up the boundary conditions and flow devices between reactors or reservoirs.
+
+4. Define a reactor network which contains all the reactors previously created.
+
+5. Advance the simulation in time, typically in a for- or while-loop. Note that only the current state is stored in Cantera by default. If you want to observe the transient states, you manually have to keep track of them.
+
+6. Analyze the data.
+
+Note that Cantera always solves a transient problem. If you are interested in steady-state conditions, you can run your simulation for a long time until the states are converged (see e.g. example `surf_pfr.py <cython/examples/reactors_surf_pfr.html>`_, `combustor.py <cython/examples/reactors_combustor.html>`_).
+
+Cantera comes with a broad variety of well-commented example scrips for reactor networks. Please refer to them for further information (`Python <cython/examples.html>`_, `Matlab <matlab/examples.html>`_).
+
+
+Common Reactor Types and their Implementation in Cantera
+========================================================
+
+
+Batch Reactor at Constant Volume or at Constant Pressure
+--------------------------------------------------------
+
+If you are interested in how a homogeneous chemical composition changes in time when it is left to its own, a simple batch reactor can be used. Two versions are commonly considered: A rigid vessel with fixed volume but variable pressure, or a system idealized at constant pressure but varying volume.
+
+In Cantera, such a simulation can be performed very easily. The initial state of the solution can be specified by composition and a set of thermodynamic parameters (like temperature and pressure) as a standard Cantera solution object. Upon its base, a general (Ideal Gas) Reactor or an (Ideal Gas) Constant Pressure Reactor can be created, depending on if a constant volume or constant pressure batch reactor should be considered, respectively. The behavior of the solution in time can be simulated as a very simple Reactor Network containing only the formerly created reactor.
+
+An example for such a Batch Reactor is `reactor1.py <cython/examples/reactors_reactor1.html>`_.
+
+
+Continuously Stirred Tank Reactor
+---------------------------------
+
+In literature, Continuously Stirred Tank Reactors (CSTR) are often used to perform basic computations like blowout limits in combustion systems. They are also often referred to as Well-Stirred Reactor (WSR), or Perfectly Stirred Reactor (PSR), or Longwell Reactor.
+
+A CSTR assumes a confined volume in which the fluid is homogeneous at all points. A stream of reactants enter the confinement at a steady mass flow rate `\dot{m}`. The reactor is assumed to be steady. Therefore, the identical mass flow rate `\dot{m}` of products have to exit the confinement. The mass contained in the confinement `m` divided by the mass flow rate `\dot{m}` defines the mean residence time of the fluid in the confinement.
```

Comment:
I think this looks pretty good. For the first sentence of the second paragraph, I might just say that "Steady state solutions to CSTRs are often of interest.".

In the last paragraph, I would avoid mentioning the radical injection method of initializing a WSR, because it seems to confuse a lot of users. Initializing the reactor with the constant HP/UV equilibrium, which corresponds to the solution as the flow rate approaches zero, is the best method in my experience. I don't know if this always will give you the reacting solution, but it usually seems to work.